.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+---
+version: 2
+
+references:
+  git_cache_key: &git_cache_key
+    git-cache-{{ .Branch }}-{{ .Revision }}
+
+  git_save_cache: &git_save_cache
+    save_cache:
+      key: *git_cache_key
+      paths:
+        - .
+
+  git_restore_cache: &git_restore_cache
+    restore_cache:
+      key: *git_cache_key
+
+  mvn_cache_key: &mvn_cache_key
+    dependency-cache-{{ checksum "project.clj" }}
+
+  mvn_save_cache: &mvn_save_cache
+    save_cache:
+      key: *mvn_cache_key
+      paths:
+        - .m2
+
+  mvn_restore_cache: &mvn_restore_cache
+    restore_cache:
+      key: *mvn_cache_key
+
+  build_config: &build_config
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/jukebox
+
+jobs:
+  checkout_code:
+    <<: *build_config
+    steps:
+      - *git_restore_cache
+      - checkout
+      - *git_save_cache
+
+  deps:
+    <<: *build_config
+    steps:
+      - *git_restore_cache
+      - *mvn_restore_cache
+      - run:
+          lein with-profiles +test deps
+      - *mvn_save_cache
+
+  build:
```

Comment:
I'd name this `test`. The only reason it's named build in jackdaw is for compatibility with the previous config and the project settings which expect the success of the "build" job.